src/Spago/Config.purs

```diff
@@ -155,7 +155,7 @@ readWorkspace { maybeSelectedPackage, pureBuild } = do
   -- First try to read the config in the root. It _has_ to contain a workspace
   -- configuration, or we fail early.
   { workspace, package: maybePackage, workspaceDoc } <- Core.readConfig "spago.yaml" >>= case _ of
-    Left err -> die [ "Couldn't parse Spago config, error:\n  " <> err, "The configuration file help can be found here https://github.com/purescript/spago#the-configuration-file" ]
+    Left err -> die [ "Couldn't parse Spago config, error:\n  " <> err, ". The configuration file help can be found here https://github.com/purescript/spago#the-configuration-file" ]
```

Comment:
Why this change? The new string will go in a new line, so this is going to look weird. Let's add a test with a fixture to verify how it looks like

Reply:
This should be reverted